### PR TITLE
Make Admin Hub Middleware Configurable

### PR DESCRIPTION
**Pull request by PHPJunior**
I made minor changes to the admin hub middleware to be configurable.

### CODE REVIEW

**Comment by glennjacobs** [comment]
@PHPJunior I think we'd want it to default if the config isn't set.

**Comment by glennjacobs** [comment]
We probably need to update the docs to include this also.

**Comment by PHPJunior** [comment]
I will update the docs too

File: docs/admin-hub/configuration.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Configuration
+
+### System
+If you wish to change the default configuration of Admin Hub's path and middleware. You can change the configuration in the `lunar-hub/system.php` file.
```
Comment (glennjacobs) [inline docs/admin-hub/configuration.md:4]: Suggest we tweak the text as follows

"If you wish to change the default configuration of the Admin Hub's path and middleware, you can change the configuration in the `lunar-hub/system.php` file."

**Review by alecritson — APPROVED** [review]

**Comment by nick-nds** [comment]
Hello,

It seems that the changes from this pull request have not been merged into the latest release version, 0.3.0. I am wondering whether these fixes are scheduled to be included in the upcoming 0.4 release or in a future patch for the 0.3 version.

**Pull request merged**